TestSupport/planaritytesting/edge_deletion_analysis.py

```diff
@@ -43,14 +44,16 @@ def __init__(self, message):
 class EdgeDeletionAnalyzer:
     """Performs edge-deletion analysis on graph(s) in a .g6 input file"""
 
-    __MAX_NUM_ERRORS = 1000
+    # Cap this at a much lower value, since EDA for N=10 takes days if this
+    # threshold is high
+    __MAX_NUM_INVALID_OKs = 5
```

Comment:
This should be changed to 500 to match the tables being checked in. Also, I think there may need to be a code tweak based on the fact that the n10 tables are getting to 501, when the max should be 500.

Reply:
We decided to bump this back up to 500 so that as we drive down the `numInvalidOK`s for the lower graph orders (currently, `N=9` has a maximum `numInvalidOK`s of 399 for `K_{3, 3}` search, whereas for `N=10` the threshold is reached for `M=17`), we can see that max value decreasing relative to the tables committed to the repository. For each bugfix, we can commit the new version of the tables with the new (preferably lower!) `numInvalidOK`s.